doc/nrf/device_guides/working_with_fem.rst

```diff
@@ -478,6 +500,17 @@ To use the Simple GPIO implementation of FEM with SKY66112-11, complete the foll
 
    The state of the other control pins should be set according to the SKY66112-11 documentation.
    See the official `SKY66112-11 page`_ for more information.
+#. On nRF53 devices, you must also apply the same devicetree node mentioned in step 1 to the network core.
+   To do so, apply the overlay to the correct network core child image by creating an overlay file named :file:`child_image/*childImageName*.overlay` in your application directory, for example :file:`child_image/multiprotocol_rpmsg.overlay`.
```

Comment:
This is not very clear. What are "the overlay" and "the correct network core child image", and what are the contents of the file that is created?

Technically, the file's name doesn't include the `child_image` part, I don't think.

Reply:
The contents of the file are described in the earlier points. The intention was to explain that the same changes in the devicetree configuration (the same contents) must also be applied to the network core. The most common way of applying such changes is through a devicetree overlay, hence "the overlay". As for the child image - there are several possible child images that might be selected, depending on the main application configuration. The possible values are listed below and the one that matches the main application configuration should be selected.